{"uid":"d7b90e9956eb594e","name":"test_fail_and_rollback","fullName":"tests.integration.test_upgrade#test_fail_and_rollback","historyId":"997774fbf319c1a457fb1c7123a70c21","time":{"start":1719194159312,"stop":1719194774117,"duration":614805},"status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 129, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/tests/integration/test_upgrade.py\", line 159, in test_fail_and_rollback\n    await ops_test.model.block_until(\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1045, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1719193122647,"stop":1719193122648,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1719193122648,"stop":1719193122789,"duration":141},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1719193122648,"stop":1719193122648,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pgb_charm_jammy","time":{"start":1719193122790,"stop":1719193122795,"duration":5},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1719194159312,"stop":1719194159312,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1719194159312,"stop":1719194159312,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 129, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/tests/integration/test_upgrade.py\", line 159, in test_fail_and_rollback\n    await ops_test.model.block_until(\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1045, in block_until\n    await utils.block_until(done,\n  File \"/home/runner/work/pgbouncer-operator/pgbouncer-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","steps":[],"attachments":[{"uid":"9650da05587c8420","name":"log","source":"9650da05587c8420.txt","type":"text/plain","size":6398}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1719194774214,"stop":1719194775326,"duration":1112},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1719194775327,"stop":1719194775631,"duration":304},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1719194775327,"stop":1719194775327,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1719194775631,"stop":1719194775631,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1719194775631,"stop":1719194775631,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1719194775631,"stop":1719194775632,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_upgrade"},{"name":"host","value":"fv-az1780-920"},{"name":"thread","value":"7018-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_upgrade"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"6c1d77acee11ea81","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":64,"unknown":1,"total":69},"items":[{"uid":"a7aa229370919c5","reportUrl":"../1765//#testresult/a7aa229370919c5","status":"passed","time":{"start":1719107994390,"stop":1719108103394,"duration":109004}},{"uid":"97476d2cbec337da","reportUrl":"../1764//#testresult/97476d2cbec337da","status":"passed","time":{"start":1719021385982,"stop":1719021495131,"duration":109149}},{"uid":"37481bd6c92067e1","reportUrl":"../1762//#testresult/37481bd6c92067e1","status":"passed","time":{"start":1718934917282,"stop":1718935023806,"duration":106524}},{"uid":"b50e0a1bda4a0f97","reportUrl":"../1758//#testresult/b50e0a1bda4a0f97","status":"passed","time":{"start":1718848705356,"stop":1718848862046,"duration":156690}},{"uid":"23c810a93627c7d7","reportUrl":"../1749//#testresult/23c810a93627c7d7","status":"passed","time":{"start":1718762400214,"stop":1718762504591,"duration":104377}},{"uid":"f3accab9fb2946de","reportUrl":"../1745//#testresult/f3accab9fb2946de","status":"passed","time":{"start":1718675849116,"stop":1718675966374,"duration":117258}},{"uid":"7218057a08f206f0","reportUrl":"../1740//#testresult/7218057a08f206f0","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1718589555238,"stop":1718590171298,"duration":616060}},{"uid":"2e06475579cd7a79","reportUrl":"../1739//#testresult/2e06475579cd7a79","status":"passed","time":{"start":1718503226388,"stop":1718503328047,"duration":101659}},{"uid":"bd5d23407157406","reportUrl":"../1738//#testresult/bd5d23407157406","status":"unknown","time":{}},{"uid":"13c496241e5a5e34","reportUrl":"../1730//#testresult/13c496241e5a5e34","status":"passed","time":{"start":1718330342327,"stop":1718330492466,"duration":150139}},{"uid":"883c685120cf5d3","reportUrl":"../1708//#testresult/883c685120cf5d3","status":"passed","time":{"start":1718158600656,"stop":1718158746516,"duration":145860}},{"uid":"1fbbc53577e83f22","reportUrl":"../1679//#testresult/1fbbc53577e83f22","status":"passed","time":{"start":1717898667296,"stop":1717898814017,"duration":146721}},{"uid":"dbe4007f038eb02f","reportUrl":"../1678//#testresult/dbe4007f038eb02f","status":"passed","time":{"start":1717812910449,"stop":1717813050021,"duration":139572}},{"uid":"5998ca322b12050d","reportUrl":"../1665//#testresult/5998ca322b12050d","status":"passed","time":{"start":1717466940089,"stop":1717467078578,"duration":138489}},{"uid":"b8d26c33b6dc398c","reportUrl":"../1659//#testresult/b8d26c33b6dc398c","status":"passed","time":{"start":1717379780054,"stop":1717379905867,"duration":125813}},{"uid":"c298609c1ddd84a4","reportUrl":"../1654//#testresult/c298609c1ddd84a4","status":"passed","time":{"start":1717207116715,"stop":1717207246051,"duration":129336}},{"uid":"5b3eacd771bdf4d5","reportUrl":"../1652//#testresult/5b3eacd771bdf4d5","status":"passed","time":{"start":1717121870914,"stop":1717122009432,"duration":138518}},{"uid":"2e6b53813d62972b","reportUrl":"../1649//#testresult/2e6b53813d62972b","status":"passed","time":{"start":1717034328577,"stop":1717034417878,"duration":89301}},{"uid":"2ef11b819c1e7053","reportUrl":"../1647//#testresult/2ef11b819c1e7053","status":"passed","time":{"start":1716947889424,"stop":1716948030175,"duration":140751}},{"uid":"82a1b6120b5e2219","reportUrl":"../1643//#testresult/82a1b6120b5e2219","status":"passed","time":{"start":1716861698043,"stop":1716861784373,"duration":86330}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"d7b90e9956eb594e.json","parameterValues":[]}